fuzz_lightyear/main.py

```diff
@@ -121,12 +124,14 @@ def run_tests(
     :param seed: used for random generation of test input
     :param ignore_exceptions: if True, ignores HTTP exceptions to requests.
     :param disable_unicode: if True, only use ASCII characters to fuzz strings
+    :param max_fuzz_depth: used for preventing recursion Objects with self reference
     """
     if seed is not None:
         get_settings().seed = seed
```

Comment:
Does this line (and the other lines where `get_settings` is called) actually do anything? `get_settings` returns `Settings()`, so my understanding here is that this line sets the value of `seed` in `Settings`, but then that `Settings` object gets lost (i.e. the `Settings` returned on L134 would not have the same value for `seed` set in here).

Reply:
@lorenzodb1 The `Settings` class is singleton here ([`@lru_cache(maxsize=1)`](https://github.com/Yelp/fuzz-lightyear/blob/master/fuzz_lightyear/settings.py#L24)). So, the values set here will persist and will be used in other parts of the project.